services/workflowbuilder/src/infra/database/prisma/repositories/prisma-procedureProject-repository.ts

```diff
@@ -17,4 +19,40 @@ export class prismaProcedureProjectRepository
       data: procedureProjectPrismaData,
     });
   }
+
+  async findByField(
+    field: string,
+    value: string,
+  ): Promise<ProcedureProject[] | null> {
+    const procedureProjectPrismaData =
+      await this.prismaService.procedureProject.findMany({
+        where: {
+          [field]: {
+            contains: value,
+          },
+        },
+      });
+    if (!procedureProjectPrismaData) {
+      throw console.log('No procedureProject where found');
```

Comment:
Instead of throw console log, you can

```suggestion
      throw new Error("Procedure Project not found")
```